.github/workflows/pr_compliance.yml

```diff
@@ -0,0 +1,21 @@
+name: PR Compliance
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened]
+
+# Action should have write permission to make updates to PR
+permissions:
+  pull-requests: write
+
+jobs:
+  pr-compliance:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mtfoley/pr-compliance-action@main
+        with:
+          ignore-authors:
+            - dependabot
+            - dependabot[bot]
+            - all-contributors[bot]
+          title-check-enable: true
```

Comment:
This means the `pull_request_title.yml` file can be deleted now, right?

Also, isn't `title-check-enable: true` the default setting?

```suggestion
```

Reply:
Yup, that's correct